src/main/java/org/opensearch/geospatial/GeospatialParser.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GeospatialParser {
```

Comment:
What's the purpose of this class, maybe you can put some javadoc description? For now I see only one method that to me is doing more on a conversion/transformation rather than parsing. Are going to add more parser related functionality to this class later?